src/components/MarketMySharesForm/index.js

```diff
@@ -87,7 +87,15 @@ class MarketMySharesForm extends Component {
   @autobind
   handleSellShare(shareId, shareAmount) {
     const shareIndex = this.props.marketShares.map(share => share.id).indexOf(shareId)
-    return this.props.sellShares(this.props.market, shareIndex, shareAmount).then(() => this.props.reset())
+    const shareBalance = new Decimal(this.props.marketShares[shareIndex].balance).div(1e18)
+    const selectedSellAmount = new Decimal(shareAmount)
+    const sellAmount = new Decimal(shareBalance)
+      .div(1e18)
+      .sub(selectedSellAmount)
```

Comment:
I think this value is not in Wei. If it is, can you change the variable name to indicate it's in wei? I think the value passed to handleSellShare is in ETH on Line 306